### DISCUSSION TITLE
Configuring Vitest coverage thresholds decimal place significance

### DISCUSSION THREAD

**Discussion by nicklascschmidt** [vitest-dev/vitest#6143] [Q&A] [answered]
When I push a new PR, even a one line fix will typically require an update to the vite config coverage thresholds because the thresholds are set to two decimal places. I have my app set up to run a pre-push hook (with Husky) and update the thresholds automatically (using `thresholdAutoUpdate`), but my issue is I run into merge conflicts for the vite config file constantly due to the coverage values changing by minute decimal places.

Is there any way to configure vitest to track the coverage values with whole numbers or specify the amount of decimal places to test against?

Other info:
I'm using an old version of `vitest @ 0.34.6`, so curious if there's an existing configuration option available in my current version, but understand if an update is required to get the functionality I'm requesting.

My vite config looks something like this:
```js
...
test: {
    ...
    coverage: {
      provider: 'v8',
      thresholdAutoUpdate: true,
      reporter: ['text', 'text-summary'],
      lines: 80.77,
      functions: 75.25,
      statements: 80.77,
      branches: 86.62,
    }
  }
```

Happy to answer any clarifying questions as needed. Thank you in advance for reading!

**Comment by AriPerkkio** [comment, marked as answer]
> Is there any way to configure vitest to track the coverage values with whole numbers or specify the amount of decimal places to test against?

No, I think the current values are originally formatted here: https://github.com/istanbuljs/istanbuljs/blob/main/packages/istanbul-lib-coverage/lib/percent.js. Vitest will then use them as is.

**Reply by nicklascschmidt** [reply]
Bummer, but appreciate the quick reply :+1:
